modules/markup/common/linkify.go

```diff
@@ -122,9 +122,7 @@ func (s *linkifyParser) Parse(parent ast.Node, block text.Reader, pc parser.Cont
 			}
 		}
 	}
-	if m == nil {
-		return nil
-	}
+
```

Comment:
Why remove these?

Reply:
because of dead code

see L100 & L110

If we like to keep it, we need a reason we should write down here ...

Reply:
@lunny do we have a reason to keep it?